src/components/Search.tsx

```diff
@@ -161,9 +161,9 @@ export const Search: FunctionComponent<SearchProps> = ({
           searchParameters={{
             // prettier-ignore
             facetFilters: [
-                'tags:docs',
-                `framework:${framework}`,
-                `version:${version}`
+                ['tags:docs', 'tags:recipes'],
```

Comment:
Use back-ticks to stay consistent with code formatting

Reply:
aye aye